src/pages/experiments/[experimentId]/data-processing/index.jsx

```diff
@@ -141,11 +171,24 @@ const DataProcessingPage = () => {
                 () => {
                   const newId = Math.min(stepId + 1, steps.length - 1);
```

Comment:
I see, so you are treating the order of the steps as their ids, so for example, cell size distribution has id = 1. I wonder how can we make this more obvious in the code

Reply:
Correct, maybe `index` or `stepIdx` would be better?

Reply:
stepIdx sounds good yes